app/code/Magento/Backend/view/adminhtml/web/js/store-switcher.js

```diff
@@ -0,0 +1,88 @@
+define([
+    'jquery',
+    'mage/translate',
+    'Magento_Ui/js/modal/confirm'
+], function(jQuery, $t, confirm) {
```

Comment:
I believe it's better to use short alias of jQuery in all places. What do you thing?
```suggestion
define([
    'jquery',
    'mage/translate',
    'Magento_Ui/js/modal/confirm'
], function($, $t, confirm) {
```